nuxeo-core/nuxeo-core-management-test/src/main/resources/core-management-tests-component.xml

```diff
@@ -31,4 +30,21 @@
       class="org.nuxeo.ecm.core.management.test.FakeDocumentStoreHandler" />
   </extension>
 
+  <implementation class="org.nuxeo.ecm.core.management.test.probes.FakeService" />
+  <service>
+    <provide interface="org.nuxeo.ecm.core.management.test.probes.FakeService" />
+  </service>
+
+  <extension point="probes"
+    target="org.nuxeo.ecm.core.management.CoreManagementComponent">
```

Comment:
target on same line than point ?

Reply:
That depends on the formatter wrapping, but we tend to write the `target` attribute first and `point` second.

Reply:
inverted for clarity 👍